src/js/components/Tiles.js

```diff
@@ -55,12 +55,27 @@ export default class Tiles extends Component {
       // grab CSS styles from DOM after component mounted
       // default to medium tile size ($tile-size = 192px)
       let minColumnWidth = 192;
-      const tile =
-        document.querySelectorAll(`.${CLASS_ROOT}__masonry-column .${TILE}`);
-      if (tile && tile.length > 0) {
-        const columnTile = window.getComputedStyle(tile[0]);
-        if (columnTile && columnTile.width) {
-          minColumnWidth = parseFloat(columnTile.width);
+      if (!this.props.fill) {
+        const tiles = findDOMNode(this.refs.tiles);
+        const tile =
+          tiles.querySelectorAll(`.${CLASS_ROOT}__masonry-column .${TILE}`);
+        if (tile && tile.length > 0) {
+          const columnTile = window.getComputedStyle(tile[0]);
+          if (columnTile && columnTile.width) {
+            minColumnWidth = parseFloat(columnTile.width);
+          }
+        }
+      } else {
+        // when fill and masonry options are true, width gets
+        // overwritten, and is no longer retrievable from css
+        const { size } = this.props;
+        if (size) {
+          if (size === 'small') {
+            minColumnWidth = 96;  // $tile-small-size
```

Comment:
This will break if $tile-small-size is changed in the SCSS or overridden by a theme. Can we instead set the width when `fill && masonry` in the CSS and then reuse the DOM introspection logic in both cases?